src/ui/public/chrome/directives/loading_indicator/loading_indicator.html

```diff
@@ -0,0 +1,7 @@
+<div
+  class="loadingIndicator"
+  ng-show="chrome.httpActive.length"
+  data-test-subj="globalLoadingIndicator"
```

Comment:
what is this about ? i mean i see you find this element by this attribute .... but why not by class, why is this better and why is this attribute named such as it is ?